.codeclimate.yml

```diff
@@ -8,6 +8,13 @@ engines:
     enabled: true
   rubocop:
     enabled: true
+    checks:
+      Rubocop/Style/AndOr:
+        enabled: false
+      Rubocop/Style/GuardClause:
+        enabled: false
+      Rubocop/Style/IfUnlessModifier:
+        enabled: false
```

Comment:
Allows us to begin enforcing these locally and allow legacy code to pass at code climate. @phirefly @yozlet @nickbristow @rememberlenny